spec/models/user_spec.rb

```diff
@@ -13,4 +13,23 @@
     it {should validate_uniqueness_of(:email)}
     it {should validate_presence_of(:name)}
   end
+  describe "instance methods" do
+    it "#leagues returns array of user's leagues" do
+      user = create(:user)
+      active_season = create(:season, :active_with_existing_users)
+      retired_season = create(:season, :retired_with_existing_users)
+
+      expected = League.all
+
+      expect(user.leagues.pluck(:id)).to eq(expected.pluck(:id))
```

Comment:
probably want to use the user full_setup trait at some point. Also not pluck but use the leagues.